src/Yaml.ts

```diff
@@ -17,6 +17,12 @@
 import * as yaml from "js-yaml";
 import * as _ from "lodash";
 
+// tslint:disable-next-line:interface-over-type-literal
+export type Options = {
+    keepArrayIndent?: false | true;
+    updateAll?: false | true;
```

Comment:
Shouldn't these just be `boolean` types?

Reply:
True.